source/pip/qsharp/_qsharp.py

```diff
@@ -838,6 +838,7 @@ def circuit(
     max_operations: Optional[int] = None,
     source_locations: bool = False,
     group_by_scope: bool = False,
+    trim: bool = False,
```

Comment:
Made my own comment outside of the code to explain why I don't like the 'trim' name.